corehq/apps/app_manager/tests/data/suite/sync_request.xml

```diff
@@ -0,0 +1,34 @@
+<partial>
+  <sync-request>
+    <post url="https://www.example.com/a/test_domain/phone/claim-case/">
+      <data key="case_id" ref="session/datum[0]/case[@id]"/>
+      <data key="case_type" ref="session/datum[0]/case/type"/>
+    </post>
+    <command id="case_lists.m0.or_something">
+      <display>
+        <text>
+          <locale id="case_search.m0"/>
+        </text>
+      </display>
+    </command>
+    <session>
+      <query url="https://www.example.com/a/test_domain/phone/search/">
```

Comment:
@kaapstorm, I think we need to add a `data` node here that fetches the case type from the session. 
From what I've understood from what @phillipm told me, the `data` nodes are sent back as get request parameters. 
i.e. something like:
```
<data key="case_type" ref="instance('session')/session/data/case_type"/>
```
Need to verify how we can get the case_type from the session, or if we just hard-code it ourselves in HQ from the information we know about the module.